six/modules/python/six.sicd/source/six_sicd.i

```diff
@@ -124,6 +124,28 @@ Data* readNITF(const std::string& pathname,
     return reinterpret_cast<Data*>(reader.interleaved(region, 0));
 }
 
+
+void crop_SICD(const std::string& inPathname,
```

Comment:
Why is this here?  Doesn't seem to do anything..?

Reply:
Lines 128-147 are defining a wrapper on cropSICD that first calls 
'''
 six::XMLControlFactory::getInstance().addCreator(
          six::DataType::COMPLEX,
	  new six::XMLControlCreatorT<six::sicd::ComplexXMLControl>());
'''